programs/util.c

```diff
@@ -115,6 +115,8 @@ int UTIL_isSameFile(const char* file1, const char* file2)
 #endif
 }
 
+#ifndef _MSC_VER
+/* Using this to distinguish named pipes */
 U32 UTIL_isFIFO(const char* infilename)
```

Comment:
minor : if `UTIL_isFIFO()` doesn't exist on Windows, its declaration within `util.h` should also be gated.